po/fr.po

```diff
@@ -24,104 +25,100 @@ msgstr ""
 #: data/ui/help-overlay.blp:10
 msgctxt "shortcut window"
 msgid "General"
-msgstr ""
+msgstr "General"
 
 #: data/ui/help-overlay.blp:13
 msgctxt "shortcut window"
 msgid "Keyboard Shortcuts"
-msgstr ""
+msgstr "Raccourcis clavier"
 
 #: data/ui/help-overlay.blp:18
 msgctxt "shortcut window"
 msgid "Quit"
-msgstr ""
+msgstr "Quitter"
 
 #: data/ui/help-overlay.blp:24
 #, fuzzy
 #| msgid "Recording view"
```

Comment:
```suggestion
```

The `fuzzy` flag means "needs review or editing" and the next line contains previous source text for reference. So, we need to remove them when we reviewed/edited, otherwise our translation won't be shown in the UI.